lib/screens/home/widgets/category_main_menu.dart

```diff
@@ -15,33 +17,40 @@ class _CategoryMainMenuState extends ConsumerState<CategoryMainMenu> {
     final gridTile = ref
         .read(homeViewModelProvider.notifier)
         .cocktailMenuTiles
-        .map((tile) => _buildGridTile(withName: tile.name))
+        .map((tile) => _buildGridTile(withName: tile.name, path: tile.path))
         .toList();
 
     return Center(
         child: GridView.count(
       primary: false,
       padding: const EdgeInsets.all(20),
-      crossAxisSpacing: 10,
+      crossAxisSpacing: 20,
       mainAxisSpacing: 10,
       crossAxisCount: 2,
       children: gridTile,
     ));
   }
 
-  Widget _buildGridTile({required String withName}) {
+  // add required String routeName
+  Widget _buildGridTile({required String withName, required String path}) {
     return GridTile(
-      child: Container(
-        padding: const EdgeInsets.all(8),
-        decoration: BoxDecoration(
-            border: Border.all(color: Colors.amberAccent, width: 5),
-            color: Colors.amber.shade100,
-            borderRadius: const BorderRadius.all(Radius.circular(20))),
-        child: Center(
-            child: Text(
-          withName,
-          textAlign: TextAlign.center,
-        )),
+      child: TextButton(
+        onPressed: () {
+          final router = AutoRouter.of(context);
+          router.pushNamed('/$path');
+        },
```

Comment:
Maybe we can add a function for this onPressed function, what if we need to add more logic when the button is pressed? we rather have it put in a function for an easier understanding